src/modules/submit.js

```diff
@@ -0,0 +1,23 @@
+const submitData = () => {
+  const form = document.getElementById('form');
+  const name = document.getElementById('name');
+  const score = document.getElementById('score');
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const myUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/aoa0Cp2AIrloXom8gGRB/scores/';
+    fetch(myUrl, {
+      method: 'POST',
+      body: JSON.stringify({
+        user: name.value,
+        score: score.value,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    score.value = '';
+    name.value = '';
+  });
+};
```

Comment:
- [ ] kindly use async await method in your submit form event  your code should look like this 
```
form.addEventListener('submit', async (e) => {
...
await fetch(....)
...
}
you can also put it in a try-catch block incase there is an error :+1: